app/models/policy_configuration.rb

```diff
@@ -12,6 +12,26 @@
 class PolicyConfiguration < ApplicationRecord
   ACADEMIC_YEAR_REGEXP = /\A20\d{2}\/20\d{2}\z/.freeze
 
+  SERVICES = [
+    {
+      routing_name: "student-loans",
+      slugs: StudentLoans::SlugSequence::SLUGS,
+      policies: [StudentLoans]
+    },
+    {
+      routing_name: "maths-and-physics",
+      slugs: MathsAndPhysics::SlugSequence::SLUGS,
+      policies: [MathsAndPhysics]
+    },
+    {
+      routing_name: "early-career-payments",
+      slugs: EarlyCareerPayments::SlugSequence::SLUGS,
```

Comment:
Not now but eventually we need to rename that slug sequence to cover both ECP and LUP

Reply:
Good point, I've added a few more TODOs to where that might be affected.